# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../wvstreams/openssl.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -urNp wvstreams-4.6.1-orig/crypto/wvx509.cc wvstreams-4.6.1/crypto/wvx509.cc
--- wvstreams-4.6.1-orig/crypto/wvx509.cc	2008-10-23 22:23:49.000000000 +0200
+++ wvstreams-4.6.1/crypto/wvx509.cc	2010-01-13 15:48:13.000000000 +0100
@@ -1157,7 +1157,7 @@ WvString WvX509::get_extension(int nid) 
         
         if (ext)
         {
-            X509V3_EXT_METHOD *method = X509V3_EXT_get(ext);
+            X509V3_EXT_METHOD *method = (X509V3_EXT_METHOD *) X509V3_EXT_get(ext);
             if (!method)
             {
                 WvDynBuf buf;

